src/components/ResultsView.tsx

```diff
@@ -395,134 +399,110 @@ const ResultsView: ResultsViewComponent = ({
         results={allProcessedResults}
       />
       <div className="relative">
-        <span
-          className="absolute top-1 right-0 z-10"
-          style={!showMenuIcons && !showInterface ? { display: "none" } : {}}
+        <div
+          className="absolute top-0 right-0 h-20 w-24 z-20"
+          // if this is set to the main div, mouse events on CellEmbed cause re-renders
+          onMouseEnter={() => {
+            if (!showInterface) setShowMenuIcons(true);
+          }}
+          onMouseLeave={() => {
+            if (!showInterface) setShowMenuIcons(false);
+          }}
         >
-          {onRefresh && (
-            <Tooltip content={"Refresh Results"}>
-              <Button icon={"refresh"} minimal onClick={() => onRefresh()} />
-            </Tooltip>
-          )}
-          <Popover
-            placement="left-end"
-            isOpen={moreMenuOpen}
-            target={
-              <Button
-                minimal
-                icon={"more"}
-                className={isMenuIconDirty ? "roamjs-item-dirty" : ""}
-              />
-            }
-            onInteraction={(next, e) => {
-              if (!e) return;
-              const target = e.target as HTMLElement;
-              if (
-                target.classList.contains("bp3-menu-item") ||
-                target.closest(".bp3-menu-item")
-              ) {
-                return;
+          <div
```

Comment:
This fixed a separate issue where any `ResultsView` mode would re-render multiple times when moving the mouse over the component.

Reply:
video showing the initial render issue and the trial of `stopPropagation()` as a fix

https://www.loom.com/share/89aef4d1bb9742249b97041fd5a73fdf